docs/src/examples/State-feedback Abstraction: PWA System.jl

```diff
@@ -0,0 +1,352 @@
+using Test     #src
+# # Example: Optimal control of a PWA System by State-feedback Abstractions
+#
+#md # [![Binder](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/State-feedback Abstraction: PWA System.ipynb)
+#md # [![nbviewer](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/State-feedback Abstraction: PWA System.ipynb)
```

Comment:
Add an empty line between badges and the rest (see html output: https://dionysos-dev.github.io/Dionysos.jl/previews/PR181/generated/State-feedback%20Abstraction:%20PWA%20System/)